src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js

```diff
@@ -19,12 +21,38 @@ export class DocumentTitle extends Component {
     }
   }
 
+  getLinkTo(relation) {
+    return relation.pid_type === 'docid'
+      ? FrontSiteRoutes.documentDetailsFor(relation.pid_value)
+      : FrontSiteRoutes.seriesDetailsFor(relation.pid_value);
+  }
+
+  relationsTitle(relationsData) {
+    return !_isEmpty(relationsData) ? (
+      <Header.Subheader>
+        {relationsData.map((rel, index) => (
+          <>
+            <Link key={rel.pid_value} to={this.getLinkTo(rel)}>
+              {rel.record_metadata.title}
+            </Link>
+            {index === relationsData.length - 1 ? ' featuring:' : ' and '}
```

Comment:
not sure I understand why we need "and"?

Reply:
Because it is a list, there can be multiple relations

Reply:
normally, a volume can belong only to one multipart monograph (but multiple serials, that's why it is modelled as a list), but for this particular case we should display only one multipart